week6/dna/dna.py

```diff
@@ -0,0 +1,66 @@
+import csv
+import sys
+
+
+def main():
+
+    if len(sys.argv) != 3:
+        sys.exit("Usage: python dna.py data.csv sequence.txt")
+
+    with open(sys.argv[1]) as file:
+        reader = csv.DictReader(file)
+        database = list(reader)
+
+    with open(sys.argv[2]) as file2:
+        sequence = file2.read()
+
+    matches = {}
+    for i in database[0]:
+        matches[i] = (longest_match(sequence, i))
+
+    suspect = 'No Match'
+    suspect_counter = 1
+
+    for i in range(len(database)):
+        for j in matches:
+
+            if str(matches[j]) == database[i][j]:
+                suspect_counter += 1
+        if suspect_counter == len(matches):
+
+            suspect = database[i]['name']
+            break
+        else:
+            suspect_counter = 1
+
+    print(suspect)
+
+    return
+
+def longest_match(sequence, subsequence):
```

Comment:
```suggestion
def longest_match(sequence: list[str], subsequence: list[str]) -> str:
```

Not sure if these are the correct types but this is an example of how to use them